### DISCUSSION TITLE
Lane quest confusing question

### DISCUSSION THREAD

**Discussion by alexcyclo** [streetcomplete/StreetComplete#5380] [Feedback]
Hi! I'm new to GitHub.
I feel that the number of car lane question is confusing (at least in french). It leads on answering if the lanes are painted which a partially other issue.
![Screenshot_20231119-101128](https://github.com/streetcomplete/StreetComplete/assets/151432737/777df948-6f56-49c9-b326-04314d97124c)

Here in the example, it's a 2x1 lane (both directions) but there is no paint.

Thks for the dvlpment of this great tool!

**Comment by matkoniecz** [comment]
It seems like translation is problematic - are you interested in fixing it? See https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app

I would be thankful as I do not understand or speak French at all (though I contributed large part of Polish translation, including fixing problems like this one).

**Reply by matkoniecz** [reply]
@alexcyclo Have you tried to edit translation? Have you managed to edit?

**Comment by alexcyclo** [comment]
I would be happy to provide some translation issue.
But i still think here is more like a minor logical problem that i misunderstood. 
Il live in a neigborhood where none of the streets have painted lanes but are large enough to let cars crossing their way. So i didnt saw that "Painted Lanes" and "the number of lanes isn't the same on each side" lead to a second question. So i was guessing the "not painted lanes" was meaning "one way lanes" which wans't correct in my case.

**Reply by Helium314** [reply]
The meaning is only that there are no painted lanes, there is no statement about how many.
See also #2553, #2299 and #3138

**Reply by alexcyclo** [reply]
I read those three threads but i'm not sure of their conclusion. Shouldn't the question be more precise then : " How many **marked** lanes does this road have for car " ? Instead of "How many car lanes does this road have" (My english isn't that good, maybe my formulation isn't good) 
+ @matkoniecz

**Reply by mnalis** [reply]
@alexcyclo Well, it seems to me it does say it clearly in English once you pick the answer:

- If you answer `No marked lanes` it does not ask for lane count at all, so no confusion there hopefully
- If you answer `Marked lanes` it additionally asks `Specify the total number of marked lanes (both directions) usable by car traffic [...]`
 
![small_Screenshot_20231126_222113_de westnordost streetcomplete](https://github.com/streetcomplete/StreetComplete/assets/156656/f5afc6fd-346d-46b1-aaf9-1d08e546ee25)

Does it say something less clear in French? 

---

As for the original (first) question `How many car lanes does this road have?`, I think it was changed once already (to add reference to cars)... Perhaps  `How many marked car lanes does this road have?` question would be clearer, but note that we try to keep it as short as possible, and it is already quite long (splits into multiple lines). It might still be an improvement, though,  but it might bring its own confusion  (_"why is it asking me about how many marked lanes are there, when the lanes are not marked at all at this road?"_)

Or perhaps we might change it completely, e.g. _"How are car lanes marked here?"_ :man_shrugging: (as actual number of lanes is only asked in some of the sub-quests, and sub-quests have their own explanation texts which can clarify what number is being asked for).

---

But the core of the confusion seems to be that this quests breaks one of StreetComplete own [tenets](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md) : 
_"⚛️ Atomic quests: Per quest, only one thing should need to be answered by the user."_ and it asks (at least) **two** separate things:
- are the lanes marked or not?
- how many lanes are there in each direction?

It might have a good reason to break that rule, but by doing that it brings the confusion that you noticed - as it is no longer simple to easily explain in one short question what exactly app wants to know.

**Reply by matkoniecz** [reply]
@alexcyclo Have you managed to fix it in French translation? (not sure should I ask other people from French community to help...)
